the test fails are related though

```
FAILED lib/galaxy_test/selenium/test_dataset_edit.py::TestHistoryPanel::test_history_dataset_update_annotation_and_info - AssertionError
FAILED lib/galaxy_test/selenium/test_uploads.py::TestUploads::test_upload_file - AssertionError: Incorrect number of items in history - expected 1, found 0
assert 0 == 1
FAILED lib/galaxy_test/selenium/test_uploads.py::TestUploads::test_upload_pasted_url_content - AssertionError: Incorrect number of items in history - expected 1, found 0
assert 0 == 1
FAILED lib/galaxy_test/selenium/test_uploads.py::TestUploads::test_upload_composite_dataset_pasted_data - AssertionError: Incorrect number of items in history - expected 1, found 0
assert 0 == 1
FAILED lib/galaxy_test/selenium/test_uploads.py::TestUploads::test_upload_simplest - AssertionError: Incorrect number of items in history - expected 1, found 0
assert 0 == 1
FAILED lib/galaxy_test/selenium/test_uploads.py::TestUploads::test_upload_pair_specify_extension - KeyError: 0

```

--- a/client/src/components/Grid/configs/histories.ts
+++ b/client/src/components/Grid/configs/histories.ts
@@ -32,6 +32,7 @@ type SortKeyLiteral = "create_time" | "name" | "update_time" | undefined;
  */
 async function getData(offset: number, limit: number, search: string, sort_by: string, sort_desc: boolean) {
     const { data, headers } = await historiesFetcher({
+        view: "summary",
         limit,
         offset,
         search,

--- a/client/src/components/Grid/configs/historiesShared.ts
+++ b/client/src/components/Grid/configs/historiesShared.ts
@@ -22,6 +22,7 @@ type SortKeyLiteral = "create_time" | "name" | "update_time" | undefined;
  */
 async function getData(offset: number, limit: number, search: string, sort_by: string, sort_desc: boolean) {
     const { data, headers } = await historiesFetcher({
+        view: "summary",
         limit,
         offset,
         search,

--- a/lib/galaxy/webapps/galaxy/api/histories.py
+++ b/lib/galaxy/webapps/galaxy/api/histories.py
@@ -191,7 +191,7 @@ class FastAPIHistories:
                 trans, serialization_params, filter_query_params, deleted_only=deleted, all_histories=all
             )
         else:
-            payload = HistoryIndexQueryPayload.construct(
+            payload = HistoryIndexQueryPayload.model_construct(
                 show_own=show_own,
                 show_published=show_published,
                 show_shared=show_shared,

--- a/lib/galaxy/webapps/galaxy/api/histories.py
+++ b/lib/galaxy/webapps/galaxy/api/histories.py
@@ -201,7 +201,9 @@ class FastAPIHistories:
                 offset=offset,
                 search=search,
             )
-            entries, total_matches = self.service.index_query(trans, payload, include_total_count=True)
+            entries, total_matches = self.service.index_query(
+                trans, payload, serialization_params, include_total_count=True
+            )
             response.headers["total_matches"] = str(total_matches)
             return entries
 

--- a/lib/galaxy/webapps/galaxy/services/histories.py
+++ b/lib/galaxy/webapps/galaxy/services/histories.py
@@ -216,6 +216,7 @@ class HistoriesService(ServiceBase, ConsumesModelStores, ServesExportStores):
         self,
         trans,
         payload: HistoryIndexQueryPayload,
+        serialization_params: SerializationParams,
         include_total_count: bool = False,
     ) -> Tuple[List[AnyHistoryView], int]:
         """Return a list of History accessible by the user
@@ -223,7 +224,6 @@ class HistoriesService(ServiceBase, ConsumesModelStores, ServesExportStores):
         :rtype:     list
         :returns:   dictionaries containing History details
         """
-        serialization_params = SerializationParams(default_view="detailed")
         entries, total_matches = self.manager.index_query(trans, payload, include_total_count)
         return (
             [self._serialize_history(trans, entry, serialization_params) for entry in entries],

